src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -336,6 +339,25 @@ public void simulateConsumeItem(@NotNull ItemStack consumable)
 							!Bukkit.isPrimaryThread());
 			Bukkit.getPluginManager().callEvent(stopConsumeEvent);
 		}
+		else
+		{
+			// apply status effects
+			FoodConsumption foodConsumption = FoodConsumption.getFor(consumable.getType());
+			for (FoodConsumption.FoodEffect foodEffect : foodConsumption.foodEffects())
+			{
+				// TODO: how to handle foodEffect with probability?
```

Comment:
Randomness and unit testing does not match well, probably a good idea to instead have a boolean switch in this method that determines the outcome.